sourcecode/hub/resources/views/components/content/search.blade.php

```diff
@@ -25,5 +22,19 @@ class="btn-secondary"
             <x-icon name="search" />
             {{ trans('messages.search') }}
         </x-form.button>
+
+        <button
+            id="filterButton"
+            class="btn btn-secondary d-lg-none d-md-none filter-button"
```

Comment:
The `d-lg-none` is not required as the `d-md-none` will hide it for resolutions 'md' and up